### ISSUE TITLE
Handle st2 pack config

### ISSUE THREAD

**Issue by mickmcgrath13** [bitovi/github-actions-deploy-stackstorm#19]
Perhaps we could provide env vars that contain pack config similar to how the docker-to-ec2 action handles DOT_ENV

**Comment by armab** [comment]
It would be natural for someone managing stackstorm if we'd allow users to create an entire dir with the pack configs, the same structure as it's in the StackStorm itself. So user creates the following dirs and files:
```
st2configs
  - aws
    - config.yaml
  - jira
    - config.yaml
```
we pick that up and copy it to the StackStorm instance, doing `st2ctl reload`.
This could be managed by Ansible too and is more portable and easy to manage from the user's perspective.

**Comment by mickmcgrath13** [comment]
right. we'd want to allow them to somehow manage secret values, too.  perhaps a yaml merge of what's checked-in to the repo (as you suggested) and an env var (yaml contents)?  That's what the deploy-docker-to-ec2 action does for `.env` files.

alternatively, users could just perform whatever logic they need to pull/set up the pack config in a previous step

**Comment by mickmcgrath13** [comment]
re ansible: Ideally, imo, users shouldn't have to be aware of ansible; only the features of stackstorm, and all the IaC is abstracted from them, so I like the idea of "provide it like ST2 would expect it" via checked-in and generated repo files

**Comment by armab** [comment]
> Ideally, imo, users shouldn't have to be aware of ansible

Yeah, I think that is worth the effort for BitOps in general considering we target beginners.

However, for StackStorm, there are more expectations and needs of control in terms of configuration. As well as the target group is more mature than just the DevOps beginners.

I think the job on abstracting the AWS was done nicely.
But looking at https://github.com/bitovi/github-actions-deploy-stackstorm/issues/7#issuecomment-1370101010, the reality IMO that we can't fully hide such an elephant in the room like Ansible configuration when deploying StackStorm.
